library/src/scala/Predef.scala

```diff
@@ -609,6 +611,11 @@ object Predef extends LowPriorityImplicits {
      */
     inline def runtimeChecked: x.type @RuntimeChecked = x: @RuntimeChecked
 
+  // extension method sugar ---------------------------------------------
+  extension[A] (self: A)
+    transparent inline def -> [B](y: B): (A, B) = (self, y)
```

Comment:
This should really be a non-transparent. If it bothers CC, then CC must adapt.

Making this `transparent inline` has a *huge* cost in terms of compatibility guarantees. That should be avoided basically at all cost.